indexer/README.md

```diff
@@ -1,2 +1,80 @@
 ## Readme for poetry
 
+## Pre processing:
+
+The pre-processing step in indexing involves parsing the file before chunking it. The function parse_file() accepts the entire pathname of the file as a parameter and uses it to extract the extension of the file. Depending on what the file extension is, the corresponding functions for parsing the file are called. Currently, the implementation for file parsing has been provided for files of the format pdf, docs, xslx and json only.
+
+-**DOCX Parser**: 
+        ```python
+        def docx_parser(docx_file_path: str):
+            return docx2txt.process(docx_file_path)```
```

Comment:
If the code is not needed, can we remove them everywhere?

Reply:
Done